packages/web-components/src/button/button.styles.ts

```diff
@@ -125,10 +125,12 @@ export const styles = css`
     fill: currentColor;
   }
 
+  [slot='start'],
```

Comment:
Curious, what does this target that `::slotted([slot='start'])` does not?

Reply:
The default slotted content, because it’s not actually slotted. We could use a class alternatively…